python/dgl/transforms/module.py

```diff
@@ -1592,6 +1592,7 @@ def __call__(self, g):
 
         for i in range(1, self.k + 1):
             g.ndata[self.out_feat_name + '_' + str(i)] = feat_list[i - 1]
+        return g
```

Comment:
I just realized that the feature transforms (as well as most of others) are in-place.